charts/helm_lib/README.md

```diff
@@ -0,0 +1,847 @@
+| [helm_lib_module_container_security_context_capabilities_drop_all_and_run_as_user_custom](#helm_lib_module_container_security_context_capabilities_drop_all_and_run_as_user_custom) |
+| **Module Storage Class** |
+| [helm_lib_module_storage_class_annotations](#helm_lib_module_storage_class_annotations) |
+| **Monitoring Grafana Dashboards** |
+| [helm_lib_grafana_dashboard_definitions_recursion](#helm_lib_grafana_dashboard_definitions_recursion) |
+| [helm_lib_grafana_dashboard_definitions](#helm_lib_grafana_dashboard_definitions) |
+| [helm_lib_single_dashboard](#helm_lib_single_dashboard) |
+| **Monitoring Prometheus Rules** |
+| [helm_lib_prometheus_rules_recursion](#helm_lib_prometheus_rules_recursion) |
+| [helm_lib_prometheus_rules](#helm_lib_prometheus_rules) |
+| [helm_lib_prometheus_target_scrape_timeout_seconds](#helm_lib_prometheus_target_scrape_timeout_seconds) |
+| **Node Affinity** |
+| [helm_lib_node_selector](#helm_lib_node_selector) |
+| [helm_lib_tolerations](#helm_lib_tolerations) |
+| **Pod Disruption Budget** |
+| [helm_lib_pdb_daemonset](#helm_lib_pdb_daemonset) |
+| **Priority Class** |
+| [helm_lib_priority_class](#helm_lib_priority_class) |
+| **Resources Management** |
+| [helm_lib_resources_management_pod_resources](#helm_lib_resources_management_pod_resources) |
+| [helm_lib_resources_management_original_pod_resources](#helm_lib_resources_management_original_pod_resources) |
+| [helm_lib_resources_management_vpa_spec](#helm_lib_resources_management_vpa_spec) |
+| [helm_lib_resources_management_cpu_units_to_millicores](#helm_lib_resources_management_cpu_units_to_millicores) |
+| [helm_lib_resources_management_memory_units_to_bytes](#helm_lib_resources_management_memory_units_to_bytes) |
+| [helm_lib_vpa_kube_rbac_proxy_resources](#helm_lib_vpa_kube_rbac_proxy_resources) |
+| [helm_lib_container_kube_rbac_proxy_resources](#helm_lib_container_kube_rbac_proxy_resources) |
+| **Spec For High Availability** |
+| [helm_lib_pod_anti_affinity_for_ha](#helm_lib_pod_anti_affinity_for_ha) |
+| [helm_lib_deployment_on_master_strategy_and_replicas_for_ha](#helm_lib_deployment_on_master_strategy_and_replicas_for_ha) |
+| [helm_lib_deployment_strategy_and_replicas_for_ha](#helm_lib_deployment_strategy_and_replicas_for_ha) |
+
+## Envs For Proxy
+
+### helm_lib_envs_for_proxy
+
+ Add HTTP_PROXY, HTTPS_PROXY and NO_PROXY environment variables for container 
+ depends on [proxy settings](https://deckhouse.io/documentation/v1/deckhouse-configure-global.html#parameters-modules-proxy) 
+
+#### Usage
+
+`{{ include "helm_lib_envs_for_proxy" . }} `
+
+#### Arguments
+
+-  Dot object (.) with .Values, .Chart, etc 
+
+## High Availability
+
+### helm_lib_is_ha_to_value
+
+ returns value <yes> if cluster is highly available, else — returns <no> 
+
+#### Usage
+
+`{{ include "helm_lib_is_ha_to_value" (list . <yes> <no>) }} `
+
+#### Arguments
+
+list:
+-  Dot object (.) with .Values, .Chart, etc 
```

Comment:
"Dot object" is just a "template context". I'd rephrase it everywhere

Reply:
fixed